alvr/server/build.rs

```diff
@@ -62,7 +62,8 @@ fn main() {
     build
         .cpp(true)
         .files(source_files_paths)
-        .flag_if_supported("-std=c++17")
+        .flag_if_supported("-std=c++20")
+        .flag_if_supported("-fdiagnostics-color=always")
```

Comment:
Could you explain why you added this isn't the default auto and that should be good enough?

Reply:
Because auto just doesn't work for some reason and I prefer looking through my walls of errors with visual clues. But I can get rid of it

Reply:
Nah just wondering